tools/libfrencutils/mosaic_util.c

```diff
@@ -62,7 +62,6 @@ void set_rotate_poly_true(void){
     void error_handler(char *str)
     error handler: will print out error message and then abort
 ***********************************************************/
-#pragma acc routine seq
 void error_handler(const char *msg)
```

Comment:
Can this function be cleaned up? 
Also, it seems that there is an "exit(1)" that can be executed but without printing anything. Is this true?

Reply:
I haven't changed the exit(1) statement.
I think this is as clean as we can get this function right now. 
We need to look up what kind of error handling OpenACC has and I was going to leave that til the end.